src/main/java/com/pusher/client/example/PrivateEncryptedChannelExampleApp.java

```diff
@@ -9,13 +9,32 @@
 import com.pusher.client.connection.ConnectionStateChange;
 import com.pusher.client.util.HttpAuthorizer;
 
+/*
+This app demonstrates how to use Private Encrypted Channels.
+
+Please ensure you update this relevant parts below with your Pusher credentials before running.
+and ensure you have set up an authorization endpoint with end to end encryption. Your Pusher credentials
+can be found at https://dashboard.pusher.com, selecting the channels project, and visiting the App Keys
+tab.
+
+A demonstration authorization endpoint using nodejs can be found
+https://github.com/pusher/pusher-channels-auth-example#using-e2e-encryption
+
+For more information on private encrypted channels please read
+https://pusher.com/docs/channels/using_channels/encrypted-channels
+
+For more pecific information on how to use private encrypted channels check out
+https://github.com/pusher/pusher-websocket-java#private-encrypted-channels
+ */
+
 public class PrivateEncryptedChannelExampleApp implements
         ConnectionEventListener, PrivateEncryptedChannelEventListener {
 
-    private String apiKey = "FILL_ME_IN"; // "key" at https://dashboard.pusher.com
+    private String key = "FILL_ME_IN";
```

Comment:
I think with so little context to the variable name, it's worth keeping at least a comment directing people to the dashboard to get one.

In general, I think `apiKey` is a pretty common term and will help people grasp quicker where they would probably find it.

Reply:
I've written in the comment at the top of the file they can get all that info from the dashboard - but i think your right the variable name should be more descriptive - maybe even `channelsKey`?